src/andromede/libs/antares_historic/antares_historic.yml

```diff
@@ -199,16 +199,16 @@ library:
         - id: inflows
           time-dependent: true
           scenario-dependent: true
+        - id: initial_level
```

Comment:
Ok, I have put in as a variable initially as in Antares I think there is a setting to choose whether it is optimized or not. For now we cannot handle it without having two different models in the modeler (or maybe put it as a variable with same lower/upper bound ?), so ok we can put it as parameter for now